### PR TITLE
Merge bug Fixes

### PR DESCRIPTION
**Pull request by jeffin07**
Fixed the remaining task in #3 

1. Fix warning by removing redundant std::move() 
2. Added double quotes around BUILD_INTERFACE and INSTALL_INTERFACE

### CODE REVIEW

**Comment by jeffin07** [comment]
I changed a file in submodule `3rd/FBoW/CMakeLists.txt` but i 'm not able to commit that file. Any suggestions?

File: src/openvslam/CMakeLists.txt [hunk 1]
```diff
@@ -129,12 +129,12 @@ endif()
...
-                           $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/src/>
-                           $<INSTALL_INTERFACE:include/openvslam/3rd/json/include>
-                           $<INSTALL_INTERFACE:include/openvslam/3rd/spdlog/include>
-                           $<INSTALL_INTERFACE:include/>)
+                           "$<BUILD_INTERFACE:${json_INCLUDE_DIR}>"
+                           "$<BUILD_INTERFACE:${spdlog_INCLUDE_DIR}>"
+                           "$<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/src/>""
+                           "$<INSTALL_INTERFACE:include/openvslam/3rd/json/include>"
+                           "$<INSTALL_INTERFACE:include/openvslam/3rd/spdlog/include>""
+                           "$<INSTALL_INTERFACE:include/>")
```
Comment (ymd-stella) [inline src/openvslam/CMakeLists.txt:137]:
```suggestion
                           "$<BUILD_INTERFACE:${json_INCLUDE_DIR}>"
                           "$<BUILD_INTERFACE:${spdlog_INCLUDE_DIR}>"
                           "$<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/src/>"
                           "$<INSTALL_INTERFACE:include/openvslam/3rd/json/include>"
                           "$<INSTALL_INTERFACE:include/openvslam/3rd/spdlog/include>"
                           "$<INSTALL_INTERFACE:include/>")
```

**Review by ymd-stella — CHANGES_REQUESTED** [review]

**Comment by ymd-stella** [comment]
Please submit a pull request to https://github.com/OpenVSLAM-Community/FBoW for b1acfd8a898b6dbe9387ae2bc25d6f27e54c7234. FBoW is a submodule.

**Comment by ymd-stella** [comment]
@jeffin07 Please submit a pull request to https://github.com/OpenVSLAM-Community/FBoW for b1acfd8. FBoW is a submodule.

**Comment by jeffin07** [comment]
@ymd-stella  so do i need to  undo the current changes in FBoW

**Comment by ymd-stella** [comment]
Use f427af71e1362cea1f35c7914136fee4d0d02adc.

**Comment by jeffin07** [comment]
@ymd-stella I'm sorry i didn't get that . Could you please explain.sorry I'm new to submodule.

**Comment by ymd-stella** [comment]
Stage the changes as follows:

```bash
cd 3rd/FBoW
git fetch
git reset --hard origin/master
cd ..
git add FBoW
```

**Comment by ymd-stella** [comment]
It doesn't seem to have changed.

**Pull request merged**